bahmnicore-api/src/main/resources/moduleApplicationContext.xml

```diff
@@ -10,10 +10,23 @@
 
     <context:component-scan base-package="org.bahmni.module"/>
 
-    <bean id="sqlSearchService" class="org.bahmni.module.bahmnicore.service.impl.SqlSearchServiceImpl">
+    <bean id="sqlSearchServiceTarget" class="org.bahmni.module.bahmnicore.service.impl.SqlSearchServiceImpl">
         <property name="administrationService" ref="adminService"/>
     </bean>
 
+    <bean id="authorizationInterceptor" class="org.openmrs.aop.AuthorizationAdvice"/>
```

Comment:
hi @rohit-yawalkar . bean `authorizationInterceptor` is already defined [here](https://github.com/openmrs/openmrs-core/blob/master/api/src/main/resources/applicationContext-service.xml#L531)

Reply:
yes @sivareddytw, change worked, there was no need to redefine it